src/components/SearchBar/SearchBar.component.tsx

```diff
@@ -0,0 +1,61 @@
+import { Search as SearchIcon } from '@mui/icons-material';
+import { Box, TextField, Typography } from '@mui/material';
+
+import { StyledAutocomplete } from '@components';
+import { type SearchBarProps } from '@components';
```

Comment:
Hey the potential to cause cyclic depenency is more. If your importing these from the barrel file. https://github.com/angular/angular-cli/issues/7369


```suggestion
import { StyledAutocomplete } from './SearchBar.styles.tsx';
import { type SearchBarProps } from './SearchBar.types.ts';
```